tkltest/generate/unit/augment.py

```diff
@@ -186,7 +186,10 @@ def __compute_base_and_augmenting_tests_coverage(ctd_test_dir, evosuite_test_dir
     # create backup of CTD-guided tests
     ctd_test_dir_bak = os.path.basename(ctd_test_dir) + '-augmentation-bak'
     shutil.rmtree(ctd_test_dir_bak, ignore_errors=True)
-    shutil.move(ctd_test_dir + os.sep + 'monolithic', ctd_test_dir_bak)
+    os.mkdir(ctd_test_dir_bak)
+    for dir in [dir for dir in os.listdir(ctd_test_dir) if os.path.isdir(dir) and dir != 'build' and dir != 'target']:
+        print(dir, ctd_test_dir_bak)
```

Comment:
seems this print statement needs to be removed - please check

Reply:
removing